app/_how-tos/encrypt-sensitive-data-in-kong-gateway-with-keyring.md

```diff
@@ -35,6 +41,127 @@ cleanup:
 min_version:
     gateway: '3.4'
 ---
-@todo
 
-based on https://docs.konghq.com/gateway/latest/kong-enterprise/db-encryption/#getting-started 
\ No newline at end of file
+## 1. Generate an RSA key pair
+
+These keys are needed for [disaster recovery](/gateway/keyring/#disaster-recovery). You can generate them using OpenSSL:
+```sh
+openssl genrsa -out private.pem 2048
+openssl rsa -in private.pem -pubout -out public.pem
+```
+
+## 2. Set environment variables
+
+Set the variables needed to start {{site.base_gateway}} with Keyring enabled:
+```sh
+export KONG_KEYRING_ENABLED=on
+export KONG_KEYRING_STRATEGY=cluster
+export KONG_KEYRING_RECOVERY_PUBLIC_KEY=$(cat public.pem | base64)
+```
+
+{:.info}
+> **Note:** `KONG_KEYRING_RECOVERY_PUBLIC_KEY` can be:
+* The absolute path to the generated public key file
+* The public key content
+* The base64-encoded public key content
+
+## 3. Start {{site.base_gateway}}
+
+Create the {{site.base_gateway}} container with the environment variables. In this example, we can use the quickstart:
+```sh
+curl -Ls https://get.konghq.com/quickstart | bash -s -- -e KONG_LICENSE_DATA \
+    -e KONG_KEYRING_ENABLED \
+    -e KONG_KEYRING_STRATEGY \
+    -e KONG_KEYRING_RECOVERY_PUBLIC_KEY
+```
+
+## 4. Generate a key
+
+Using the Admin API, generate a new key in the Keyring:
+{% control_plane_request %}
+  url: /keyring/generate
+  method: POST
+  headers:
+      - 'Accept: application/json'
+      - 'Kong-Admin-Token: $KONG_ADMIN_TOKEN'
```

Comment:
I didn't have RBAC enabled in my test env, so this Kong Admin Token header wasn't necessary. Do you need RBAC to use keyring? If not, then I'd remove it.

```suggestion
```